notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<p>It would be helpful to print the best score somewhere for reference! I didn't find the actual number anywhere in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Well the scoring metric has been changing a lot too so maybe not so comparable. But yeah once we've settled on something it'd be good to be able to compare across time and efforts. We could even compete. :grimmace:</p>